### DISCUSSION TITLE
Project Theme: Refreshing - Can I Make Updates to Themes I Created?

### DISCUSSION THREAD

**Discussion by nunniii** [JanDeDobbeleer/oh-my-posh#4196] [General]
Hope everyone is doing well! I wanted to have a chat about the theme of our project. I know we can't create new official project themes, but I'd like to know if I can give a makeover to one of the themes I've created. I think the changes turned out nicely

![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/69170710/ce8fd491-5280-4401-92aa-45b1af963afb)
that’s the original:
https://github.com/JanDeDobbeleer/oh-my-posh/pull/787

and and improved the theme for this:
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/69170710/8d3b81fc-c849-4c55-80de-2de5f7c31bd0)

**Comment by nunniii** [comment]
@JanDeDobbeleer 😄

**Reply by JanDeDobbeleer** [reply]
Yes you can

**Reply by nunniii** [reply]
okay thanks, I'll make a pull request 🦕⭐
